include/clap/ext/params.h

```diff
@@ -279,16 +279,13 @@ typedef struct clap_host_params {
    // [main-thread]
    void (*clear)(const clap_host_t *host, clap_id param_id, clap_param_clear_flags flags);
 
-   // Request the host to call clap_plugin_params->fush().
-   // This is useful if the plugin has parameters value changes to report to the host but the plugin
-   // is not processing.
+   // Request a parameter flush.
    //
-   // eg. the plugin has a USB socket to some hardware controllers and receives a parameter change
-   // while it is not processing.
+   // If the plugin is processing, then the parameters flush will happen during the process() call.
+   // If the plugin isn't processing, then the host will call to clap_plugin_params->fush().
```

Comment:
flush, not fush

I think this is better as

```
Request a parameter flush.

If the plugin is processing, this will result in no action. The process call
will run normally. If plugin isn't processing, the host will make a subsequent
call to clap_plugin_params->flush(). As a result, this function is always
safe to call from a non-audio thread (typically the UI thread on a gesture)
whether processing is active or not.
```

Reply:
Done, thank you for the text :+1: